pkg/kubelet/cm/containermap/container_map.go

```diff
@@ -36,11 +37,7 @@ func NewContainerMap() ContainerMap {
 
 // Clone creates a deep copy of the ContainerMap
 func (cm ContainerMap) Clone() ContainerMap {
```

Comment:
personally I'd drop this function entirely as it just uses a stdlib function anyway

Reply:
I thought about it. I'd argue that if we bothered to create a type around essentially a map, we should not break the abstraction, and if so we need the trivial `Clone` method.

Other (more far-fetching?) options:
- dissolve `ContainerMap` and make use an actual map + helper functions
- make `ContainerMap` more opaque, fully wrap it into a struct, to open the option in the future to add locks (spoiler: https://github.com/kubernetes/kubernetes/pull/128657 was caused largely because I incorrectly remembered `ContainerMap` is more than such a thin wrapper. I confused it with `pod_devices`)

I'm open to conversation here.